web/src/reducers/activities.test.js

```diff
@@ -589,343 +589,24 @@ describe('activities reducer', () => {
-                      entries: [
-                        { year: '2018', amount: 10 },
-                        { year: '2019', amount: 20 }
-                      ]
-                    },
-                    {
-                      id: 'e2',
-                      category: 'category 2',
-                      description: 'desc 2',
-                      entries: [
-                        { year: '2018', amount: 30 },
-                        { year: '2019', amount: 40 }
-                      ]
-                    }
-                  ],
-                  standardsAndConditions: {
-                    businessResults: 'biz results',
-                    documentation: 'docs',
-                    industryStandards: 'standards',
-                    interoperability: 'interop',
-                    keyPersonnel: 'locksmiths',
-                    leverage: 'lever',
-                    modularity: 'lego blocks',
-                    minimizeCost: 'save money',
-                    mita: 'meeta',
-                    mitigationStrategy: 'run away',
-                    reporting: 'moop moop'
-                  },
-                  quarterlyFFP: [
-                    {
-                      q1: {
-                        combined: 0.2,
-                        contractors: 0.3,
-                        state: 0.4
-                      },
-                      q2: {
-                        combined: 0.4,
-                        contractors: 0.3,
-                        state: 0.2
-                      },
-                      q3: {
-                        combined: 0.3,
-                        contractors: 0.2,
-                        state: 0.4
-                      },
-                      q4: {
-                        combined: 0.3,
-                        contractors: 0.4,
-                        state: 0.2
-                      },
-                      year: 2018
-                    },
-                    {
-                      q1: {
-                        combined: 0.25,
-                        contractors: 0.25,
-                        state: 0.25
-                      },
-                      q2: {
-                        combined: 0.25,
-                        contractors: 0.25,
-                        state: 0.25
-                      },
-                      q3: {
-                        combined: 0.25,
-                        contractors: 0.25,
-                        state: 0.25
-                      },
-                      q4: {
-                        combined: 0.25,
-                        contractors: 0.25,
-                        state: 0.25
-                      },
-                      year: 2019
-                    }
-                  ]
-                }
-              ],
-              years: ['2018', '2019']
+                { key: 'a' },
+                { key: 'b', name: 'Program Administration' },
+                { key: 'c' }
+              ]
```

Comment:
Because we're just copying activities, that makes this test a lot simpler.  :)

Reply:
🙌